addons/mail/static/src/discuss/core/common/action_panel.scss

```diff
@@ -0,0 +1,3 @@
+.o-mail-ActionPanel-back-button {
```

Comment:
`back-button` => `backButton` (just to follow convention of classnames `o-mail-ComponentName-partName`, so we can very quickly identify part name at a glance)

Reply:
done :+1: